hobby-blog.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang = "en" >
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Halisa personal portfolio website</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <!-- <link rel="stylesheet" type="text/css" href="css/color.css" /> -->
+</head> 
+<body>
+    <main>
+        <header>
+            <nav class="navbar">
+                <ul>
+                    <li class="current-page">My Hobbies</li>
+                    <li><a href="about.html">About Me</a></li>
+                    <li><a href="portfolio.html">My Portfolio</a></li>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="contact.html">Contact Me</a></li>
+                </ul> 
+            </nav>
+        </header> 
+        <section id="hobbies">
+            <h1 class="headings">MY HOBBIES</h1>
+            <div class="my_hobbies">
+                <div class="box">
+                    <img src="img/workout.jpeg" alt="">
+                    <h1 class="headings">Work out</h1>
+                    <p>I love working out.</p>
+                </div>
+                <div class="box">
+                    <img src="img/travel.jpeg" alt="">
+                    <h1 class="headings">Travel</h1>
+                    <p>I love traveling.</p>
+                </div>
+                <div class="box">
+                    <img src="img/reading.jpeg" alt="">
+                    <h1 class="headings">Reading</h1>
+                    <p>I love reading.</p>
+                </div>
+            </div>
+        </section>
+        <footer>
+            <p>&copy; 2021 Halisa Saipulla - Ada Developers Academy C15</p>
+        </footer>
+</body> 
```

Comment:
Another missing `main`

```suggestion
            <p>&copy; 2021 Halisa Saipulla - Ada Developers Academy C15</p>
        </footer>
      </main>
</body> 
```